internal/common/baseclient/basepresident.go

```diff
@@ -58,7 +58,7 @@ func (p *BasePresident) SetTaxationAmount(islandsResources map[shared.ClientID]s
 
 	for clientID, clientReport := range islandsResources {
 		if clientReport.Reported {
-			taxAmountMap[clientID] = shared.Resources(float64(clientReport.ReportedAmount) * rand.Float64())
+			taxAmountMap[clientID] = shared.Resources(float64(clientReport.ReportedAmount) * (rand.Float64() * 0.1 + 0.1))
```

Comment:
base client changes should be in separate PR?

Reply:
also indeed, moving to another PR